### PR TITLE
Added Dutch translation

### PR DESCRIPTION
**Pull request by Vistaus**
🙂

### CODE REVIEW

File: locale/nl/LC_MESSAGES/rclone-manager.po [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+msgstr "Hoofd-aankoppellocatie"
+
+msgid "Path where the remote disks are mounted"
+msgstr "De locatie waar externe schijven worden aangekoppeld"
+
+msgid "Filenames to be ignored"
+msgstr "Te negeren bestandsnamen"
+
+msgid "Comma separed filenames to be ignored by inotify"
+msgstr ""
```
Comment (germanztz) [inline locale/nl/LC_MESSAGES/rclone-manager.po:97]: msgstr "Een kommagescheiden lijst met bestandsnamen die dienen te worden genegeerd door inotify"

File: locale/nl/LC_MESSAGES/rclone-manager.po [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+"Een kommagescheiden lijst met bestandsnamen die dienen te worden genegeerd"
+" door inotify"
+
+msgid "Terminal command"
+msgstr "Terminalvenster"
+
+msgid ""
```
Comment (germanztz) [inline locale/nl/LC_MESSAGES/rclone-manager.po:104]: msgid "Command to call a new terminal window in order to perform rclone manager actions"

File: locale/nl/LC_MESSAGES/rclone-manager.po [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+"Command to call a new terminal window in order to perform rclone manager"
+" actions"
+msgstr ""
```
Comment (germanztz) [inline locale/nl/LC_MESSAGES/rclone-manager.po:107]: msgstr "De opdracht om een nieuw terminalvenster te openen om Rclone-acties te kunnen uitvoeren"

File: locale/nl/LC_MESSAGES/rclone-manager.po [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+"De opdracht om een nieuw terminalvenster te openen om Rclone-acties te kunnen"
+" uitvoeren"
+
+msgid "File browser command"
+msgstr "Bestandsbeheerder"
+
+msgid ""
```
Comment (germanztz) [inline locale/nl/LC_MESSAGES/rclone-manager.po:114]: msgid "Command to call a new file browser window in order to explore configuration folders"

File: locale/nl/LC_MESSAGES/rclone-manager.po [hunk 5]
```diff
@@ -0,0 +1,199 @@
...
+"Command to call a new file browser window in order to explore configuration"
+" folders"
+msgstr ""
```
Comment (germanztz) [inline locale/nl/LC_MESSAGES/rclone-manager.po:117]: msgstr "De opdracht om een nieuw bestandsbeheervenster te openen om de configuratiemappen te kunnen verkennen"

**Review by germanztz — CHANGES_REQUESTED** [review]
Hi Heimen, Thanks a lot for you translation!!
I don't know if the multiple lines are on purpose, please confirm or fix.

You're the first person is using the extension, I think. feel free to open any issue for bugs etc

Thanks

**Comment by Vistaus** [comment]
Hmm, I have no idea what happened. I used Lokalize to translate and I never had this multiple line issue before. :/ 

Will fix this tomorrow! 🙂

**Comment by Vistaus** [comment]
Still no idea what happened, but it should be fixed now! 🙂

**Comment by Vistaus** [comment]
Cool, thanks!

And I'll translate the new lines when needed. 🙂

**Pull request merged**